components/automate-gateway/handler/iam/v2beta/rules/rules.go

```diff
@@ -141,6 +141,22 @@ func (s *Server) ApplyRulesCancel(ctx context.Context,
 	return &pb_resp.ApplyRulesCancelResp{}, nil
 }
 
+func (s *Server) ApplyRulesStatus(ctx context.Context,
+	req *pb_req.ApplyRulesStatusReq) (*pb_resp.ApplyRulesStatusResp, error) {
```

Comment:
Delete the param name `req` as it is not used. (Same comment Stephan had left for me in last PR; I corrected this in authz but I guess I did not check the gateway corollary.)
Please make the same change in `ApplyRulesStart` and `ApplyRulesCancel`, too (not shown here).

Reply:
Surprised that didn't set of any linter alarm bells. Is the gateway linted at all? If not :sadpanda:.